### PR TITLE
Add page title to pages in Console app using React Helmet

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose
Note $subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx [hunk 1]
```diff
@@ -404,6 +404,7 @@ const OIDCScopesEditPage: FunctionComponent<RouteComponentProps<OIDCScopesEditPa
 
         return (
             <PageLayout
+                pageTitle={"Edit " + scope.displayName}
```
Comment (brionmario) [inline apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx:407]:
```suggestion
                pageTitle={ "Edit " + scope.displayName }
```

File: modules/react-components/src/layouts/page.tsx [hunk 2]
```diff
@@ -86,6 +92,11 @@ export const PageLayout: FunctionComponent<PropsWithChildren<PageLayoutPropsInte
     );
 
     return (
+        <>
+        <Helmet>
+            <meta charSet="utf-8" />
+            <title>{pageTitle}</title>
```
Comment (brionmario) [inline modules/react-components/src/layouts/page.tsx:98]:
```suggestion
            <title>{ pageTitle }</title>
```

File: modules/react-components/src/layouts/page.tsx [hunk 3]
```diff
@@ -21,6 +21,7 @@ import classNames from "classnames";
 import React, { FunctionComponent, PropsWithChildren, ReactElement } from "react";
 import { Divider } from "semantic-ui-react";
 import { PageHeader, PageHeaderPropsInterface } from "../components";
+import {Helmet} from "react-helmet"
```
Comment (brionmario) [inline modules/react-components/src/layouts/page.tsx:24]: Check if `ESLint` works.

```suggestion
import { Helmet } from "react-helmet"
```

File: modules/react-components/src/layouts/page.tsx [hunk 4]
```diff
@@ -86,6 +92,11 @@ export const PageLayout: FunctionComponent<PropsWithChildren<PageLayoutPropsInte
     );
 
     return (
+        <>
+        <Helmet>
```
Comment (brionmario) [inline modules/react-components/src/layouts/page.tsx:96]:
```suggestion
            <Helmet>
```

File: modules/react-components/src/layouts/page.tsx (hunk 3 above)
Comment (pavinduLakshan) [reply modules/react-components/src/layouts/page.tsx:24]: will check

**Review by brionmario — APPROVED** [review]

File: modules/react-components/src/layouts/page.tsx (hunk 4 above)
Comment (pavinduLakshan) [reply modules/react-components/src/layouts/page.tsx]: fixed with 3461a60

File: modules/react-components/src/layouts/page.tsx [hunk 5]
```diff
@@ -40,6 +41,10 @@ export interface PageLayoutPropsInterface extends PageHeaderPropsInterface, Test
      * Flag to enable/disable padding.
      */
     padded?: boolean;
+    /**
+     * page name to display in the browser tab
```
Comment (Rashmini) [inline modules/react-components/src/layouts/page.tsx:45]:
```suggestion
     * Page name to display in the browser tab.
```

Comment (pavinduLakshan) [reply modules/react-components/src/layouts/page.tsx:45]: Fixed in https://github.com/wso2/identity-apps/pull/3438

File: modules/react-components/src/layouts/page.tsx [hunk 6]
```diff
@@ -21,6 +21,7 @@ import classNames from "classnames";
 import React, { FunctionComponent, PropsWithChildren, ReactElement } from "react";
 import { Divider } from "semantic-ui-react";
 import { PageHeader, PageHeaderPropsInterface } from "../components";
+import { Helmet } from "react-helmet"
```
Comment (NipuniBhagya) [inline modules/react-components/src/layouts/page.tsx:24]: Add a semicolon at the end of the statement.

File: modules/react-components/src/layouts/page.tsx [hunk 7]
```diff
@@ -91,19 +97,24 @@ export const PageLayout: FunctionComponent<PropsWithChildren<PageLayoutPropsInte
...
+            <div className={layoutClasses} data-testid={testId} data-componentid={componentId}>
+                <div className={layoutContentClasses}>
+                    { componentAbovePageHeader && componentAbovePageHeader }
+                    <PageHeader
+                        action={action}
+                        data-testid={`${testId}-page-header`}
+                        data-componentid={`${componentId}-page-header`}
+                        {...rest}
+                    />
+                    {contentTopMargin && <Divider hidden />}
```
Comment (NipuniBhagya) [inline modules/react-components/src/layouts/page.tsx:113]: Fix the indentation and add spaces before and after curly brackets.

**Pull request merged**